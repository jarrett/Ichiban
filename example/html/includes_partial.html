<p>Here comes the partial:</p>

<%= partial '_partial.html' %>