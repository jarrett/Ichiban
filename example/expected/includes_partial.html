<!DOCTYPE html>
<html>
  <head>
    <title>Example Site</title>
  </head>
  
  <body>
    <h1>Example Site</h1>
    
    <p>Here comes the partial:</p>

<p>Version 2. Current path: /includes_partial/</p>
  </body>
</html>